tests/functional/adapter/constraints/test_constraints.py

```diff
@@ -27,14 +27,14 @@
 
 _expected_sql_trino = """
 create table <model_identifier> (
-    id integer not null,
+    "id" integer not null,
```

Comment:
@damian3031 is applying the `quote` property for field having constraints in any way related to the changes from this PR?

Reply:
I see it has been addressed as well on https://github.com/starburstdata/dbt-trino/pull/315

Reply:
it's not related